scripts/pi-hole/php/teleporter.php

```diff
@@ -148,7 +148,7 @@ function getWildcardListContent() {
 else
 {
 	$filename = "pi-hole-teleporter_".date("Y-m-d_h-i-s").".zip";
-	$archive_file_name = "/var/www/html/".$filename;
+	$archive_file_name = "/etc/pihole/".$filename;
```

Comment:
Yeah, that was my original motivation for putting where it is right now - this was the only directory which I found across the OSs we support which had write access for the web server (also across web servers, e.g. `apache2`).